sections/debt/components/PortfolioTable/PortfolioTable.tsx

```diff
@@ -127,10 +127,16 @@ const DebtPoolTable: FC<DebtPoolTableProps> = ({
 				Header: <>{t('synths.assets.synths.table.asset')}</>,
 				accessor: 'currencyKey',
 				Cell: (cellProps: CellProps<CryptoBalance, CryptoBalance['currencyKey']>) => {
+					let displayName = cellProps.value;
+					if (cellProps.value === Synths.sETH) {
+						displayName = CryptoCurrency.ETH;
+					} else if (cellProps.value === Synths.sBTC) {
+						displayName = CryptoCurrency.BTC;
+					}
 					return (
 						<Legend>
-							<Currency.Name currencyKey={cellProps.value} showIcon={true} />
-							{(cellProps.value === Synths.sETH || cellProps.value === Synths.sBTC) && (
+							<Currency.Name currencyKey={displayName} showIcon={true} />
+							{(displayName === CryptoCurrency.ETH || displayName === CryptoCurrency.BTC) && (
```

Comment:
@mi-yu I've pushed a fix here because the tooltip wasn't showing anymore. I also updated the copy.